main.js

```diff
@@ -133,6 +152,28 @@ define(function (require, exports, module) {
         };
     }
     
+    function _applyColors() {
+        var userColors = _preferences.get("colors"),
+            template = _.template(stylesTemplate),
+            css = template({
+              "whitespace": userColors.light.whitespace,
```

Comment:
For whatever reason, this one key is not being rendered in both light and dark, but all the others work. I've checked the resulting CSS and it is all valid, so it's odd why it is not working.